push-down-test/run-tests.sh

```diff
@@ -193,11 +221,18 @@ function clean_build() {
 function kill_all_proc() {
   echo
   echo "+ Kill all processes"
-
-  pkill -9 -U ${USER} tidb-server
-  pkill -9 -U ${USER} tikv-server
-  pkill -9 -U ${USER} pd-server
-  exit 0
+  echo "  - Running processes"
+  ps ux | grep tidb-server
+  ps ux | grep tikv-server
+  ps ux | grep pd-server
+  echo "  - Killing processes"
+  killall -9 tidb-server
```

Comment:
Killing other users' instances may not be a good idea.

Reply:
$USER is not available in our Jenkins

Reply:
Then this test can not run in a dev environment that contains other users' instances.

Reply:
You can run without `clean` to bypass this restriction

Reply:
Actually in our own IDC environment, users are not separated by users, instead a shared user is used, so that previous way also does not work well..

Reply:
So the prime aim is to make test run well in Jenkins. Running in dev env needs some extra work... Well, it's ok...